tests/integration/integration-test.py

```diff
@@ -46,8 +47,9 @@ def test_get_notes_path(self, endpoint='/notes'):
             actual_note_id = response_data['id']
             self.assertEqual(actual_note_id, note_id)
 
+        invalid_notes_id = random.sample(range(930000000, 939999999), 5)
```

Comment:
Naming issue: should be `invalid_notes_ids`. 

This format is also misleading. The notes id is constructed like this: `<9-digit student ID>-<random string>`. The way you generate the invalid note ids is more like random students ids.

Reply:
the note id is the student id - random string, so a student id without a random string is invalid here.
do you suggest that i make them completely random and not in the format of a student id?